evil-visual-mark-mode.el

```diff
@@ -42,7 +42,7 @@
   :prefix "evil-visual-mark-mode"
   :group 'evil)
 
-(defcustom evil-visual-mark-exclude-marks '()
+(defcustom evil-visual-mark-exclude-marks '("^" "[" "]")
```

Comment:
I agree string is better than characters, but we should also change the custom type from `'(repeat character)` to `'(repeat string)` .